tests/tests.js

```diff
@@ -56,9 +56,9 @@ module.exports = {
     'Test From Style String': function (test) {
         var style = new cssstyle.CSSStyleDeclaration();
         test.expect(8);
-        style.cssText = 'color: blue; background-color: red; width: 78%';
-        test.ok(3 === style.length, 'length is not 3');
-        test.ok('color: blue; background-color: red; width: 78%;' === style.cssText, 'cssText is wrong');
+        style.cssText = 'color: blue; background-color: red; width: 78%; height: 50vh;';
+        test.ok(4 === style.length, 'length is not 4');
+        test.ok('color: blue; background-color: red; width: 78%; height: 50vh;' === style.cssText, 'cssText is wrong');
```

Comment:
I don't know what the reasoning behind the original code was here but the assertion names seem wrong here too. 

This assertion name is incorrect "length is not 3"...

```
test.ok(3 === style.length, 'length is not 3');
```

It should be 

```
test.ok(3 === style.length, 'length is 3');
```
🤔


 

Reply:
Ah, yes that would actually make sense as to why its worded as a failure message. The docs for nodeunit don't really specify so it might have just been the preference of the original author and maintainers. 

In that case, lets just leave it with the style of the other tests until we can swap out the test runner to something that is not deprecated.

Reply:
If you have a particular testing library you like, I'd be willing to help make that switch.

Reply:
In other projects I use jest or mocha/chai. Would like to use either of these.

I have an issue for this here https://github.com/jsakas/CSSStyleDeclaration/issues/65